stylesheets/app.scss

```diff
@@ -23,6 +23,8 @@
 
     .channel-messages {
       @include span-columns(8 of 10);
+      max-height: 200px;
+      overflow: auto;
```

Comment:
haven't pulled this branch to look at it, but isn't 200px like super tiny? am i crazy?

also, let's try to get jack's PR #34 merged first, then reapply these changes to the newly-reorganized CSS!